hypervisor/arch/x86/timer.c

```diff
@@ -286,3 +277,16 @@ void calibrate_tsc(void)
 	tsc_khz = (uint32_t)(tsc_hz / 1000UL);
 	printf("%s, tsc_khz=%lu\n", __func__, tsc_khz);
 }
+
+void check_tsc(void)
```

Comment:
This patch is not necessary is general.

But the API name check_tsc is pretty confusing... It is kind of disable_rdtsc from user level.  HV doesn't have user level, so this code is a kind of "dead code" actually.

Either rename it, or removed it is fine to me.